.github/workflows/docker-publish.yml

```diff
@@ -33,7 +33,9 @@ env:
   # Use docker.io for Docker Hub if empty
   REGISTRY: ghcr.io
   # github.repository as <account>/<repo>
-  IMAGE_NAME: ${{github.repository_owner}}/registry-server
+  REGISTRY_SERVER: ${{github.repository_owner}}/registry-server
+  # github.repository as <account>/<repo>
+  REGISTRY_ENVOY: ${{github.repository_owner}}/registry-envoy
```

Comment:
I'm not clear on why we need to build a container for Envoy instead of directly using an existing Envoy container - in Kubernetes setups, I've been able to just replace envoy.yaml with a ConfigMap. Is this to allow Envoy to be configured with environment variables instead?

Reply:
Yes that is correct. 
Additionally the envoy container needs a proto.pb (which is build in the first stage of the Dockerfile) for the envoy proxy to work.

Reply:
Agreed about the compiled protos - I've also added that with a ConfigMap. Let's go ahead with this and add a README later that explains how/why to use this.

Reply:
(A concern that I have about this is that it is tied to a specific release of Envoy, but using this is certainly optional - people who know Envoy well can configure their own containers)